scripts/brc4/brc4_species.pl

```diff
@@ -78,13 +80,25 @@
     $a->{'BRC4.component'} cmp $b->{'BRC4.component'}
       or $a->{'species.scientific_name'} cmp $b->{'species.scientific_name'}
       or $a->{'BRC4.organism_abbrev'} cmp $b->{'BRC4.organism_abbrev'}
-  } @genomes) {
+  } @genomes){
```

Comment:
```suggestion
  } @genomes) {
```
To avoid getting blame for no actual changes.